manual/manual/html_processing/src/process_manual.ml

```diff
@@ -0,0 +1,465 @@
+(* ------------ Ocaml Web-manual -------------- *)
+
+(* Copyright San Vu Ngoc, 2020
+
+   file: process_api.ml
+
+   Post-processing the HTML of the OCaml Manual.
+
+   (The "API" side is treated by process_api.ml) *)
+
+open Soup
+open Printf
+open Common
+
+(* How the main index.html page will be called: *)
+let index_title = "Home"
+
+(* Alternative formats for the manual: *)
+let archives =
+  ["refman-html.tar.gz"; "refman.txt"; "refman.pdf"; "refman.info.tar.gz"]
+
+(* Remove number: "Chapter 1  The core language" ==> "The core language" *)
+let remove_number s =
+  Re.Str.(global_replace (regexp ".+  ") "" s)
+
+let toc_get_title li =
+  let r = Re.Str.regexp ".+  " in
```

Comment:
this the `remove_number` regexp, isn't it?

Reply:
good catch